purchase_partner_selectable_option/readme/DESCRIPTION.rst

```diff
@@ -1 +1,3 @@
 This module limits the partners to be selected in the purchases orders according to the "Selectable in orders" field.
+By default, this field is set to True, allowing all partners to be selectable. The default setting can be modified
+by adjusting the system parameter with the key 'purchase_partner_selectable_option.default_purchase_selectable'. 
```

Comment:
I suggest we put this in CONFIGURE.rst.

Reply:
I guess it's non-blocking as there seems to be no comment on this in the PR for the sales side https://github.com/OCA/sale-workflow/pull/2922.

Reply:
On second thought, we should clearly mention that the param should be `"0"` to make the default false.